### PR TITLE
Fix popup width and improve close button

### PR DESCRIPTION
**Pull request by AlecRust**
### Current Problem

- The popup width doesn't match the YouTube sidebar width when displayed in-page.
- The "Close Popup" link is very small and easily missed. I would expect a close icon top right, similar to what is mentioned in #594.
- The "You've saved" and "You've skipped" paragraphs aren't on their own line (small regression in #1189) ([screenshot](https://i.imgur.com/VqeXMmS.png)).

### Proposed Solution

- Allow width of popup to fill container when displayed in-page.
- Move the "Close Popup" button to the top right of the popup as an icon.
- Use `display: block` instead of `display: unset` for showing "You've saved" and "You've skipped" paragraphs.

### Screenshots

Before             |  After
:-------------------------:|:-------------------------:
image  |  image

- [x] I agree to license my contribution under LGPL-3.0 **or** my contribution is from another project with a license compatible with LGPL-3.0

To test this pull request, follow the [instructions in the wiki](https://github.com/ajayyy/SponsorBlock/wiki/Testing-a-Pull-Request).

***

### CODE REVIEW

**Comment by AlecRust** [comment]
Apologies for the whitespace changes in the diff applied by my editor. I recommend viewing with this enabled:
image

**Review by its0x08 — APPROVED** [review]

File: src/content.ts [hunk 1]
```diff
@@ -1638,17 +1638,22 @@ function openInfoMenu() {
...
+            const closeButtonIcon = document.createElement("img");
+            closeButtonIcon.src = chrome.extension.getURL("icons/close.png");
+            closeButtonIcon.width = 16;
+            closeButtonIcon.height = 16;
+            closeButton.appendChild(closeButtonIcon);
+            closeButton.setAttribute("title", chrome.i18n.getMessage("closePopup"));
+            closeButton.classList.add("sbCloseButton");
             closeButton.setAttribute("align", "center");
             closeButton.addEventListener("click", closeInfoMenu);
             // Theme based color
```
Comment (ajayyy) [inline src/content.ts:1651]: Probability should not change colour based on theme anymore

**Comment by ajayyy** [comment]
Will have to test on ultra-wide displays, but I remember it appearing pretty ugly there

**Comment by AlecRust** [comment]
My resolution is 3440×1440, seems ok. But more testing the better!

File: src/content.ts (hunk 1 above)
Comment (AlecRust) [reply src/content.ts:1651]: Is this a change for this PR?

Comment (ajayyy) [reply src/content.ts:1651]: Yes, otherwise the close button will be white on light theme, even though it is inside the SB popup which is always dark

Comment (AlecRust) [reply src/content.ts]: Thanks, I've removed this.

**Comment by mchangrh** [comment]
LGTM on 2560x1080 Ultrawide

Theater + default modes both look fine

**Pull request merged**